frontend/src/metabase/notifications/NotificationsActionsMenu/DashboardSubscriptionMenuItem.unit.spec.tsx

```diff
@@ -0,0 +1,115 @@
+import { setupNotificationChannelsEndpoints } from "__support__/server-mocks/pulse";
+import { renderWithProviders, screen } from "__support__/ui";
+import { Menu } from "metabase/ui";
+import type { ChannelApiResponse } from "metabase-types/api";
+import {
+  createMockCard,
+  createMockDashboard,
+  createMockDashboardCard,
+  createMockUser,
+} from "metabase-types/api/mocks";
+import { createMockState } from "metabase-types/store/mocks";
+
+import { DashboardSubscriptionMenuItem } from "./DashboardSubscriptionMenuItem";
+
+const mockOnClick = jest.fn();
+
+const createDashboardWithDataCards = () => {
+  return createMockDashboard({
+    dashcards: [
+      createMockDashboardCard({
+        card: createMockCard({ display: "table" }),
+      }),
+    ],
+  });
+};
+
+const setup = ({
+  isSuperuser = false,
+  hasEmailSetup = false,
+  hasSlackSetup = false,
+  dashboard: mockDashboard = createDashboardWithDataCards(),
+}: {
+  isSuperuser?: boolean;
+  hasEmailSetup?: boolean;
+  hasSlackSetup?: boolean;
+  dashboard?: ReturnType<typeof createMockDashboard>;
+} = {}) => {
+  const currentUser = createMockUser({
+    is_superuser: isSuperuser,
+  });
+  const storeInitialState = createMockState({
+    currentUser,
+  });
+
+  setupNotificationChannelsEndpoints({
+    email: { configured: hasEmailSetup },
+    slack: { configured: hasSlackSetup },
+  } as ChannelApiResponse["channels"]);
+
+  renderWithProviders(
+    <Menu opened>
+      <Menu.Dropdown>
+        <DashboardSubscriptionMenuItem
+          onClick={mockOnClick}
+          dashboard={mockDashboard}
+        />
+      </Menu.Dropdown>
+    </Menu>,
+    { storeInitialState },
+  );
+};
+
+describe("DashboardSubscriptionMenuItem", () => {
+  describe("when user can manage subscriptions and dashboard has data cards", () => {
+    it("should show 'Subscriptions' menu item when email and slack are not set up", async () => {
```

Comment:
nit: if a describe block only has one test in it, you probably don't need to keep the describe